test/libsolidity/semanticTests/inlineAssembly/inline_assembly_embedded_function_call.sol

```diff
@@ -3,9 +3,9 @@ contract C {
         assembly {
             let d:= 0x10
 
-            function asmfun(a, b, c) - > x, y, z {
```

Comment:
These things are gone, so probably the PR needs to be rebased.

Reply:
Perhaps also merge #9619 before that for having octal numbers as a test case.

Reply:
But you did add explicit ``- >`` tests in the PR removing those, didn't you? Then I need to add back the grammar workarounds for them I guess...

Reply:
Yes, there is a single explicit test case for it.

Reply:
Added a workaround with a TODO comment to the lexer (the grammar tests should in fact fail, once we disallow it and add a test for it). I hid it away for the documentation, though, since it's a bug anyways.